constraints.txt

```diff
@@ -97,6 +99,7 @@ hdfs==2.5.8
 holoviews==1.14.1
 html5lib==1.1
 httplib2==0.17.4
+hypothesis==6.17.4
```

Comment:
- [ ] Mozilla Public license v2 is approved for apache 2, but need to confirm approval for MIT

Reply:
MPL v2 is compatible with MIT. Let's merge this PR and then relicense to Apache 2.